lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1653,6 +1669,255 @@ def mock_certificate_downloadable_status(
         }
 
 
+# pylint: disable=protected-access, no-member
+@ddt.ddt
+class ProgressPageShowCorrectnessTests(ProgressPageBaseTests):
+    """
+    Tests that verify that the progress page works correctly when displaying subsections where correctness is hidden.
+    """
+    NOW = datetime.now(UTC)
+    DAY_DELTA = timedelta(days=1)
+    YESTERDAY = NOW - DAY_DELTA
+    TOMORROW = NOW + DAY_DELTA
```

Comment:
Since `TODAY` is in the past, it behaves just like `YESTERDAY`.

Addressed with https://github.com/edx/edx-platform/pull/15018/commits/d7e44c50acc6dc3a73fc400f410cfe4ceaca834d#diff-8aefe502dfa8ee5a3e18c9227e61b034R1682, https://github.com/edx/edx-platform/pull/15018/commits/d7e44c50acc6dc3a73fc400f410cfe4ceaca834d#diff-8aefe502dfa8ee5a3e18c9227e61b034R1827, https://github.com/edx/edx-platform/pull/15018/commits/d7e44c50acc6dc3a73fc400f410cfe4ceaca834d#diff-8aefe502dfa8ee5a3e18c9227e61b034R1827, etc.